index.html

```diff
@@ -10,8 +10,35 @@
     <header>
         <nav>
             <div class="nav__hamburger-wrapper">
-                <img class="nav__hamburger" src="images/Hamburger.png" alt="Hamburger butten image">
-                
+                <div id="nav__hamburger">
+                    <img id="nav__hamburger-closeIcon" class="nav__hamburger-menu--off" src="images/close_icon.png" alt="Close icon image">
+                    <img id="nav__hamburger-hamburgerIcon" class="nav__hamburger-menu--on" src="images/Hamburger.png" alt="Hamburger butten image">
+                </div>
+                <div id="nav__hamburger-menu" class="nav__hamburger-menu--off">
+                    <div class="hamburger-menu__search-bar">
+                        <div class="hamburger-menu__search-bar__input-field">
+                            <input type="text" name="search" autocomplete="off" placeholder="Search..">
+                        </div>
+                        <div class="hamburger-menu__search-bar__image">
+                        <img class="hamburger-menu__search-image" src="images/search_icon.png" alt="Search icon">
+                        </div>
+                    </div>
+                    <div class="hamburger-menu__button" ><a href="#">HOME</a></div>
+                    <div class="hamburger-menu__button" ><a href="#">ABOUT</a></div>
+                    <div class="hamburger-menu__button" ><a href="#">MENU</a></div>
+                    <div class="hamburger-menu__button" ><a href="#">RESERVATIONS</a></div>
+                    <div class="hamburger-menu__button" ><a href="#">BLOG</a></div>
+                    <div class="hamburger-menu__submenu">
+                        <div class="hamburger-menu__submenu--title">
+                            <p>FEATURES</p>
```

Comment:
Are you sure that tag `<p>` which is used for paragraphs is the best one for this label?